_locales/en/messages.json

```
@@ -188,6 +191,9 @@
   "betaTooltip": {
     "message": "Keep in mind that this addon might have bugs."
   },
+  "bugfixTooltip": {
+    "message": "This addon fix Scratch's bugs!"
```

```suggestion
    "message": "This addon fix Scratch's bugs!"
```
    "message": "This addon fixes a Scratch bug."